TheMovieDb/Common/UIImageView+extension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIImageView+extension.swift
+//  TheMovieDb
+//
+//  Created by Ricardo Ramirez on 23/11/21.
+//
+
+import UIKit
+
+extension UIImageView {
+    func load(url: URL, completion: ((UIImage) -> Void)?) {
+        DispatchQueue.global().async { [weak self] in
+            if let data = try? Data(contentsOf: url) {
+                if let image = UIImage(data: data) {
+                    DispatchQueue.main.async {
+                        self?.image = image
+                        completion?(image)
+                    }
+                }
+            }
+        }
+    }
```

Comment:
In this case you are using two threads why?

Reply:
I use global to request data trough the Data object and main to update the image on image view